membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -96,13 +96,14 @@ class AttributeController(
       onNotFound: Result,
       sendAttributesIfNotFound: Boolean = false,
       requiredScopes: List[AccessScope],
+      metricName: String,
   ) = {
     AuthAndBackendViaAuthLibAction(requiredScopes).async { implicit request =>
       if (endpointDescription == "membership" || endpointDescription == "features") {
         DeprecatedRequestLogger.logDeprecatedRequest(request)
       }
 
-      request.user match {
+      metrics.measureDuration(metricName)(request.user match {
```

Comment:
Does this give us number of requests as well as the duration of the requests?

Reply:
Yes, it sends 2 metrics.